apis/configwatch_test.go

```diff
@@ -0,0 +1,165 @@
+// Copyright Contributors to the L3AF Project.
```

Comment:
Shall we add a unit test for the function toLowerCaseASCII?  The function got implicitly run during other unit tests.  A separate unit test shall cover more code paths.

Reply:
As Jason mentioned during the TSC meeting, it was the standard GO implementation, so we don't need our own unit test.